security/voters.rst

```diff
@@ -105,7 +105,7 @@ code like this:
             public function show(Post $post): Response
             {
                 // check for "view" access: calls all voters
-                $this->denyAccessUnlessGranted('view', $post);
+                $this->denyAccessUnlessGranted('PostVoter::VIEW', $post);
```

Comment:
```suggestion
                $this->denyAccessUnlessGranted(PostVoter::VIEW, $post);
```

Reply:
I did this change while merging. Thanks.